Add age and country to enteric fever vignette

kristinawlai opened the pull request:
Added country overall and age category by country. @kaiemjoy will this work for now?

kristinawlai committed:
updates

kristinawlai committed:
Update with country and age category

kaiemjoy reviewed (changes requested):
Looks great, thanks Kristina. I few small requests to make before merging: 
1. let's take out the age&site specific estimates for now and move them over to the posit project we are prepping, "EF Seroincidence Estimates"
3. Can you rank the bar graph from highest to lowest? (or lowest to highest also fine)

d-morrison reviewed (commented):
Is this PR back in draft-mode?

kristinawlai committed:
remove age-country for now

kristinawlai committed:
Merge branch 'main' into age-country-EntericFever

d-morrison commented on vignettes/articles/enteric_fever_example.Rmd:
```diff
@@ -262,14 +299,14 @@ ggplot(est2df, aes(y=catchment, x=incidence.rate*1000, fill=catchment)) +
 
 ## Conclusions
-In our data, we find that the overall estimated seroincidence of enteric fever in Pakistan is 153 per 1000 person-years (95% CI: 139, 169). When stratified by catchment area, we find that area KGH has a higher incidence rate than area AKU [204 per 1000 person-years (95% CI: 176, 237) vs. 125 per 1000 person-years (95% CI: 109, 143)]. 
+We find that Bangladesh has the highest overall seroincidence of enteric fever with a rate of `r round(est_countrydf$incidence.rate[1] * 1000)` per 1000 person-years, as well as the highest seroincidence by age category. In comparison, Nepal has a seroincidence rate over `r round((est_countrydf$incidence.rate[1])/(est_countrydf$incidence.rate[2]))` times lower than that of Bangladesh (`r round(es ...
```
Please make each sentence a separate line in the .Rmd file; it's a lot easier to track changes and provide feedback that way.

d-morrison replied:
Please pre-calculate the numbers that you want to include inline; for example, in the chunk above, `round((est_countrydf$incidence.rate[1])/(est_countrydf$incidence.rate[2]))` could be replaced by `rate_ratio_nepal_bangla` which would be calculated in an earlier R chunk. (this is also for reviewability and transparency of calculations to the readers).

d-morrison commented on vignettes/articles/enteric_fever_example.Rmd:
```diff
@@ -262,14 +299,14 @@ ggplot(est2df, aes(y=catchment, x=incidence.rate*1000, fill=catchment)) +
         x="Seroincience rate per 1000 person-years",
```
there's a typo here ("Seroincience")

d-morrison commented on vignettes/articles/enteric_fever_example.Rmd:
```diff
@@ -262,14 +299,14 @@ ggplot(est2df, aes(y=catchment, x=incidence.rate*1000, fill=catchment)) +
-  theme_bw() +
+  theme_linedraw() +
   theme(axis.text.y = element_text(size=11),
         axis.text.x = element_text(size=11))
```
Has this plot been turned into a function yet? If so, please use it; otherwise, please create the function! Same with any other plots that take more than ~5 lines of construction.

d-morrison reviewed (changes requested):
a few recommended changes below!

kristinawlai committed:
fix typo and in-line code

kaiemjoy committed:
Merge branch 'main' into age-country-EntericFever

kaiemjoy committed:
KA notes on vignettes, fixed scrub typhus title, reference article was cited twice

kaiemjoy committed:
color changes, expanded acknowledgments

kaiemjoy committed:
added funding section

kaiemjoy committed:
remove space

kaiemjoy committed:
a misplaced (

kaiemjoy committed:
label comma

kristinawlai committed:
Fixed typo

kristinawlai committed:
Add note about the results differing from the publication

kristinawlai committed:
Add note about the results differing from the publication

kristinawlai committed:
Add note on differences from publication

Merge branch 'age-country-EntericFever' of https://github.com/UCD-SERG/serocalculator into age-country-EntericFever

# Conflicts:
#	vignettes/articles/enteric_fever_example.Rmd

kristinawlai committed:
remove ggfortify from imports
